lib/assets/components/explore_tiles.dart

```diff
@@ -139,6 +142,7 @@ abstract class BaseExploreActionTile extends StatelessWidget {
   final ListCause cause;
   final String timeText;
   final bool completed;
+  final double aspectRatio;
```

Comment:
Adding option to change the aspect ratio of the action tiles. This is because they appear longer in the places with the stack "needs completing" stack. We therefore probably need something in the viewmodel so that the page "knows" what to set the aspect ratio to.